packages/common/observability/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@dxos/observability",
+  "version": "0.3.1",
+  "description": "Provides a common interface for app and platform observability",
+  "homepage": "https://dxos.org",
+  "bugs": "https://github.com/dxos/dxos/issues",
+  "license": "MIT",
+  "author": "DXOS.org",
+  "exports": {
+    ".": {
+      "browser": "./dist/lib/browser/index.mjs",
+      "import": "./dist/lib/browser/index.mjs",
+      "require": "./dist/lib/node/index.cjs",
+      "node": "./dist/lib/node/index.cjs"
+    },
+    "./datadog": {
+      "browser": "./dist/lib/browser/datadog/index.mjs",
+      "import": "./dist/lib/browser/datadog/index.mjs",
+      "require": "./dist/lib/node/datadog/index.cjs",
+      "node": "./dist/lib/node/datadog/index.cjs"
+    },
+    "./segment": {
+      "browser": "./dist/lib/browser/segment/index.mjs",
+      "import": "./dist/lib/browser/segment/index.mjs",
+      "require": "./dist/lib/node/segment/index.cjs",
+      "node": "./dist/lib/node/segment/index.cjs"
+    }
+  },
+  "browser": {
+    "./src/datadog/node.ts": "./src/datadog/browser.ts",
+    "./src/segment/node.ts": "./src/segment/browser.ts"
+  },
+  "types": "dist/types/src/index.d.ts",
+  "typesVersions": {
+    "*": {
+      "datadog": [
+        "dist/types/src/datadog/index.d.ts"
+      ]
+    }
+  },
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "api-keys": "node ./scripts/inject_api_keys.js"
+  },
+  "dependencies": {
+    "@dxos/async": "workspace:*",
+    "@dxos/client-protocol": "workspace:*",
+    "@dxos/client-services": "workspace:*",
+    "@dxos/config": "workspace:*",
```

Comment:
Understood. Feels a bit weird to consider it SDK since at the moment it's possible, but a bit hard to imagine others using this, but it's what feels like the right thing for a provider/SaaS independent interface.